<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtDMXClass</class>
 <widget class="QMainWindow" name="QtDMXClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtDMX</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="spinBoxDMXModes_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>510</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDMXModes_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>488</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelDMXMode">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode DMX :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdressLamp1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse Lampe 1 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGreenColor">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>G</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDMXModes">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>12</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDMXModes_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>9</number>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>508</number>
    </property>
    <property name="singleStep">
     <number>4</number>
    </property>
    <property name="pageStep">
     <number>4</number>
    </property>
    <property name="sliderPosition">
     <number>1</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelRedColor">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdressLamp3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse Lampe 3 :</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSliderColorGreen">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelPilotageLampe">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pilotage Lampe</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWhiteColor">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>200</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>W</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSliderColorBlue">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSliderIntensity">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSliderColorRed">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdressLamp2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse Lampe 2 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBlueColor">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLampValue">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelValueRed">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>10</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelValueGreen">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelValueWhite">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelValueBlue">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Changement de couleur</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtDMX.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtDMXClass</receiver>
   <slot>onSliderValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderColorRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtDMXClass</receiver>
   <slot>onSliderColorRedChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderColorGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtDMXClass</receiver>
   <slot>onSliderColorGreenChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderIntensity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtDMXClass</receiver>
   <slot>onSliderColorWhiteChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>680</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderColorBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QtDMXClass</receiver>
   <slot>onSliderColorBlueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSliderValueChanged(int)</slot>
  <slot>onSliderColorRedChanged(int)</slot>
  <slot>onSliderColorGreenChanged(int)</slot>
  <slot>onSliderColorBlueChanged(int)</slot>
  <slot>onSliderColorWhiteChanged(int)</slot>
 </slots>
</ui>
